Sources/Operators.swift

```diff
@@ -204,6 +245,18 @@ public prefix func -(offset: UIOffset) -> UIOffset {
     return UIOffset(horizontal: -offset.horizontal, vertical: -offset.vertical)
 }
 
+public func +(lhs: CGSize, rhs: UIOffset) -> CGSize {
+    return CGSize(width: lhs.width + rhs.horizontal, height: lhs.height + rhs.vertical)
+}
+
+public func +(lhs: UIOffset, rhs: CGSize) -> CGSize {
+    return CGSize(width: rhs.width + lhs.horizontal, height: rhs.height + lhs.vertical)
+}
+
+public func -(lhs: CGSize, rhs: UIOffset) -> CGSize {
+    return CGSize(width: lhs.width - rhs.horizontal, height: lhs.height - rhs.vertical)
+}
+
```

Comment:
I agree it's probably not the common behavior (although I can imagine a layout where you may want to use it), but either way I think it's unexpected (missing) behavior to provide `CGSize + UIOffset`, `UIOffset + CGSize`, and `CGSize - UIOffset`, but not `UIOffset - CGSize`.